pkg/operator/staticpod/controller/missingstaticpod/missing_static_pod_controller.go

```diff
@@ -28,6 +28,9 @@ type missingStaticPodController struct {
 	configMapListerForTargetNamespace corelisterv1.ConfigMapNamespaceLister
 	targetNamespace                   string
 	operandName                       string
+
+	lastEventEmittedForRevision int
```

Comment:
I think should keep a state by `node`, i.e. `map[string]time.Time`

Reply:
plus some clean-up logic that removes entries for new and/or successful revisions

Reply:
so maybe we need to keep by `node and revision`